src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const totalCosts = days * 40;
+
+  if (totalCosts >= 280) {
```

Comment:
The discount logic is based on `totalCosts`, but according to the task description, the discount should be applied based on the number of days rented. Consider changing the condition to check the number of days instead.